dev-lua/lua-openssl/files/lua-openssl-0.4.1-Makefile.patch

```diff
@@ -0,0 +1,51 @@
+diff --git a/Makefile b/Makefile
```

Comment:
Well, for our needs the patch will work. However, for upstream I'd improve it a bit. In particular:

1. Don't put CFLAGS in CC, fix the whole thing to respect all variables properly ;-).
2. Don't append to variables like CFLAGS, it won't work if you do like `make CFLAGS='-foo'`, I think. Instead, use extra variables.

But that's future suggestions in case you decide to do sth about it.